x-pack/platform/plugins/shared/fleet/server/services/setup/fleet_synced_integrations.ts

```diff
@@ -116,3 +126,67 @@ async function createIndex(esClient: ElasticsearchClient, indexName: string, ind
     }
   }
 }
+
+export async function createCCSIndexPatterns(
+  esClient: ElasticsearchClient,
+  savedObjectsClient: SavedObjectsClientContract,
+  savedObjectsImporter: ISavedObjectsImporter
+) {
+  const { enableSyncIntegrationsOnRemote } = appContextService.getExperimentalFeatures();
+
+  if (!enableSyncIntegrationsOnRemote) {
+    return;
+  }
+
+  if (appContextService.getConfig()?.enableManagedLogsAndMetricsDataviews !== true) {
```

Comment:
only creating index patterns if `enableManagedLogsAndMetricsDataviews` is enabled, see https://github.com/elastic/kibana/pull/205268